src/components/Progress/ProgressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { TProgressBarPropTypes } from './types'
+import {Progress} from './Progress';
+import classnames from 'classnames';
+
+export const ProgressBar = (props: TProgressBarPropTypes): JSX.Element => {
+  const {
+    stepItems,
+    size = 'large',
+    className = '',
+  } = props
+
+  return (
+    <div className={classnames('progress-bar-wrapper', className)}>
+      {stepItems?.map(({percent }, index) => (
```

Comment:
and when you will make `stepItems` mandatory you will not need `?` after `stepItems`